test/e2e_env/kubernetes/meshtimeout/api.go

```diff
@@ -0,0 +1,77 @@
+package mestimeout
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/test/e2e_env/kubernetes/env"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func API() {
+	meshName := "meshtimeout-api"
+
+	BeforeAll(func() {
+		err := NewClusterSetup().
+			Install(MeshKubernetes(meshName)).
+			Setup(env.Cluster)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	E2EAfterEach(func() {
+		Expect(
+			k8s.RunKubectlE(env.Cluster.GetTesting(), env.Cluster.GetKubectlOptions(), "delete", "meshtimeouts", "-A", "--all"),
+		).To(Succeed())
+	})
+
+	E2EAfterAll(func() {
+		Expect(env.Cluster.DeleteMesh(meshName)).To(Succeed())
+	})
+
+	It("should create MeshTimeout policy", func() {
```

Comment:
do we need this test if we're not actually testing the behaviour?

Reply:
rewritten test